pkg/process/keb_processing.go

```diff
@@ -0,0 +1,207 @@
+package process
+
+import (
+	"strings"
+	"time"
+
+	kebruntime "github.com/kyma-project/kyma-environment-broker/common/runtime"
+	"github.com/patrickmn/go-cache"
+
+	kmccache "github.com/kyma-project/kyma-metrics-collector/pkg/cache"
+	log "github.com/kyma-project/kyma-metrics-collector/pkg/logger"
+)
+
+// pollKEBForRuntimes polls KEB for runtimes information.
+func (p *Process) pollKEBForRuntimes() {
+	kebReq, err := p.KEBClient.NewRequest()
+	if err != nil {
+		p.namedLogger().With(log.KeyResult, log.ValueFail).With(log.KeyError, err.Error()).
+			Fatal("create a new request for KEB")
+	}
+
+	for {
+		runtimesPage, err := p.KEBClient.GetAllRuntimes(kebReq)
+		if err != nil {
+			p.namedLogger().With(log.KeyResult, log.ValueFail).With(log.KeyError, err.Error()).
+				Error("get runtimes from KEB")
+			time.Sleep(p.KEBClient.Config.PollWaitDuration)
+
+			continue
+		}
+
+		p.namedLogger().Debugf("num of runtimes are: %d", runtimesPage.Count)
+		p.populateCacheAndQueue(runtimesPage)
+		p.namedLogger().Debugf("length of the cache after KEB is done populating: %d", p.Cache.ItemCount())
+		p.namedLogger().Infof("waiting to poll KEB again after %v....", p.KEBClient.Config.PollWaitDuration)
```

Comment:
We are also sleeping in case of an error, but only logging here. Can we make it more consistent?